spec/models/post_spec.rb

```diff
@@ -12,11 +12,12 @@
     describe "with no replies" do
       let!(:old_tagged_at) { post.tagged_at }
 
-      it "should update edited_at and tagged_at when edited" do
+      it "should update tagged_at but not edited_at when content edited" do
         Timecop.freeze(time) do
-          post.update!(content: 'new content')
+          post.written.update!(content: 'new content')
```

Comment:
When updating a post, we still do it directly on the post contents, don't we? Should this instead check that the `post.written` update is synchronized with the `post` update?

(The change to `edited_at` here is because of the change to which model got updated, right?)